src/App/ComponentsDocumentation/components/RadioButton/index.js

```diff
@@ -37,7 +37,7 @@ const HowTo = () => (
 );
 
 const Overview = () => (
-    <>
```

Comment:
Content guidelines radio button example has the same ID as the one under error state